inference-engine/src/transformations/src/transformations/common_optimizations/add_fake_quantize_fusion.cpp

```diff
@@ -51,8 +62,8 @@ ngraph::pass::AddFakeQuantizeFusion::AddFakeQuantizeFusion() {
             const_shape.insert(const_shape.begin(), fq->get_input_partial_shape(0).rank().get_length() - const_shape.size(), 1);
             add_const = std::make_shared<opset5::Reshape>(add_const, op::Constant::create(element::u64, Shape{const_shape.size()}, const_shape), false);
         }
-        auto new_input_low = std::make_shared<opset5::Subtract>(fq->input_value(1), add_const);
-        auto new_input_high = std::make_shared<opset5::Subtract>(fq->input_value(2), add_const);
+        auto new_input_low = get_constant_from_source(std::make_shared<opset5::Subtract>(fq->input_value(1), add_const));
```

Comment:
You need to check the result of `get_constant_from_source` as it returns nullptr in some cases.